lib/game.rb

```diff
@@ -52,6 +52,7 @@ def main_menu_user_input
     if choice == "p"
       set_player1
       set_bot
+      "p"
     elsif choice == "q"
```

Comment:
Maybe add a function that lets the user quit at any time with "Q" instead of a column. We can mention that at the start:` "Press "p" to play or "q" at anytime to quit the game."`